src/Storage/Repository/PgApplicationRepository.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Common;
+using System.Drawing;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Interface.Models;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Npgsql;
+using NpgsqlTypes;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Handles applicationMetadata repository. Notice that the all methods should modify the Id attribute of the
+    /// Application, since cosmosDb fails if Id contains slashes '/'.
```

Comment:
This summary was copied over from the other implementation.

Reply:
Corrected